4.covariates.ipynb

```diff
@@ -840,10 +840,10 @@
        "TCGA-02-0047-01           0.0           0.0          0.0          0.0     0.0   \n",
        "TCGA-02-0055-01           0.0           0.0          0.0          0.0     1.0   \n",
        "\n",
-       "                 male  alive  dead  has_not_recurred  has_recurred  \n",
-       "sample_id                                                           \n",
-       "TCGA-02-0047-01   1.0    0.0   1.0               0.0           1.0  \n",
-       "TCGA-02-0055-01   0.0    0.0   1.0               0.0           1.0  \n",
+       "                 male  dead_0.0  dead_1.0  has_not_recurred  has_recurred  \n",
```

Comment:
why did these variables change? Does it have to do with the float conversion? I could envision this potentially breaking something downstream

Reply:
> I could envision this potentially breaking something downstream

Certainly! We need to have stable names.